pkgs/development/python-modules/regex/default.nix

```diff
@@ -13,11 +13,16 @@ buildPythonPackage rec {
     sha256 = "2353c0e983c4029caf32016f1dddef623c3117ac282a818468c6d2f5d541698d";
   };
 
+  postCheck = ''
+    echo "We now run tests ourselves, since the setuptools installer doesn't."
+    python -c 'import test_regex; test_regex.test_main();'
```

Comment:
`${python.interpreter}`

Reply:
Just tried to fix this, but it seems the python attribute is not in scope. (undefined variable 'python'). Should python be included in the parameter list?